### DISCUSSION TITLE
File diff output should better make trailing whitespace visible

### DISCUSSION THREAD

**Discussion by DDvO** [community/community#42792] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Some OpenSSL conventions and tools are picky on trailing whitespace.

When viewing git logs and diffs in the shell, they are clearly marked bright red,
while, e.g., at the changed line 237 on https://github.com/openssl/openssl/pull/19882/files 
there is a new extra space character after the word `down` at line end, yet it is not shown at all.

So reviewers did not notice it, and a fix was needed later: https://github.com/openssl/openssl/pull/19960 

Would be good if the GitHub diff output also highlights any whitespace at the end of changed lines.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @DDvO ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
